doc/faq.md

```diff
@@ -176,6 +176,29 @@ re-install Convos:
 
 Please [contact us](/doc/#get-in-touch) if the above instructions do not work.
 
+## Why convos does not restart after an upgrade of convos code?
```

Comment:
I think this heading sounds disconnected with the rest of the text.

Reply:
No solution for this...